frontend/src/services/terminal-service.ts

```diff
@@ -10,11 +10,3 @@ export function getGitHubTokenCommand(gitHubToken: string) {
   const event = getTerminalCommand(command, true);
   return event;
 }
-
-export function getCloneRepoCommand(gitHubToken: string, repository: string) {
```

Comment:
I think we should move the `export GITHUB_TOKEN` piece as well for consistency. Ideally we can delete this whole file

Reply:
This is used here: https://github.com/All-Hands-AI/OpenHands/blob/6ee9028d4aa2fc50eb00b7a9051e639df7c3613f/frontend/src/routes/_oh.app/hooks/use-handle-runtime-active.ts#L53-L58

We probably shouldn't move this because it is useful for someone trying to change the github token in the middle of a session. I wonder how often this happens though.

Reply:
> because it is useful for someone trying to change the github token in the middle of a session. I wonder how often this happens though.

Hah. I needed that last night, on the website (app.all-hands.dev). It did fail, though, anyway. I eventually had to give up and restart the session.

Reply:
OK fair, let's leave it for now. Ideally this would still trigger something on the backend though, rather than the frontend